crates/attestation/src/report_data.rs

```diff
@@ -99,7 +96,7 @@ pub enum ReportData {
 }
 
 impl ReportData {
-    pub fn new(tls_public_key: near_sdk::PublicKey) -> Self {
+    pub fn new(tls_public_key: [u8; 32]) -> Self {
```

Comment:
I’d prefer to have a self-explanatory strong type here rather than a raw byte array. TLS public keys are used so widely across our codebase that we should have a single type representing them, easily convertible to all the necessary formats.